### PR TITLE
Lots of V7 fixes

### LINKED ISSUE #2239: [v7] <Message deleted> is not clickable to reveal the message
As title, in previous version it was clickable to reveal the message when show/remove deleted messages were both unticked.

### PR DESCRIPTION
**Pull request by EhsanKia**
Fixes 1, 2, 3 and 5 from #2247 (one left)
Fixes 2, 6 and 9 from #2246 (four left)
Fix 1 and 2 from #2244 (done)
Fix #2239

### CODE REVIEW

File: src/modules/chat_deleted_messages/index.js [hunk 1]
```diff
@@ -78,6 +81,15 @@ class ChatDeletedMessagesModule {
         tmiRoom.on('clearchat', onClearChat);
         tmiRoom._bttvClearChatMonkeyPatched = true;
     }
+
+    checkDeleted($el, msgObj) {
```
Comment (night) [inline src/modules/chat_deleted_messages/index.js]: 1. this breaks emojis
2. turn off deleted links tbh.. there's a boolean somewhere iirc

File: src/modules/host_button/index.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ class HostButtonModule {
 
     embedHostButton() {
         if ($('#bttv-host-button').length) return;
-        $hostButton = $('<button><span></span></button>');
+        $hostButton = $('<button><span>Host</span></button>');
```
Comment (night) [inline src/modules/host_button/index.js]: not needed.. if the update fails I dont want the button to show at all

File: src/modules/notification_sound/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            defaultValue: false,
+        });
+
+        settings.on('changed.highlightFeedback', () => this.load());
+        this.load();
+    }
+
+    load() {
+        if (settings.get('highlightFeedback') !== true) return;
+        this.sound = new Audio('https://cdn.betterttv.net/assets/sounds/ts-tink.ogg');
```
Comment (night) [inline src/modules/notification_sound/index.js]: no hardcoding links, we have a way to generate a cdn url

File: src/modules/notification_sound/index.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    play(ignoreFocus = false) {
+        if (!ignoreFocus && document.hasFocus()) return;
+        if (settings.get('highlightFeedback') !== true) return;
+        console.log(this.sound.paused);
```
Comment (night) [inline src/modules/notification_sound/index.js]: uh

File: src/modules/host_button/index.js [hunk 5]
```diff
@@ -65,16 +65,19 @@ class HostButtonModule {
...
-
-        tmiSession._tmiApi
-            .get('/hosts', {host: userId})
-            .then(data => {
-                if (!data.hosts || !data.hosts.length) return;
-                hosting = data.hosts[0].target_id === channelId;
-                this.updateHostButtonText();
-            });
+        let counter = 0;
+        const interval = setInterval(() => {
```
Comment (night) [inline src/modules/host_button/index.js]: hacky solution, I'd rather load this on chat load and check if it's the channel page

File: src/modules/global_css/index.js [hunk 6]
```diff
@@ -96,6 +99,16 @@ class GlobalCSSModule {
     toggleLeftSideChat() {
         $('body').toggleClass('swap-chat', settings.get('leftSideChat'));
     }
+
+    attachTheatreListener() {
+        const playerService = App.__container__.lookup('service:persistentPlayer');
+        if (!playerService || !playerService.playerComponent || !playerService.playerComponent.player) return;
+        playerService.playerComponent.player.addEventListener('theatrechange', state => {
+            if (settings.get('darkenedMode') === true) return;
```
Comment (night) [inline src/modules/global_css/index.js]: you are not handling the case where the user exits theater mode.. we need to disable darkened mode

File: src/modules/chat_deleted_messages/index.js [hunk 7]
```diff
@@ -40,6 +36,12 @@ function onClearChat(name, tags) {
                     $unlinked.text($link.attr('href'));
                     $link.replaceWith($unlinked);
                 });
+
+                if (!settings.get('showDeletedMessages')) {
+                    const $messageClone = $message.clone();
+                    $message.click(() => $message.replaceWith($messageClone));
+                    $message.text('<message deleted>');
```
Comment (night) [inline src/modules/chat_deleted_messages/index.js]: need to make this linkified .. i.e. on hover underline .. greyed out text by enabling our deleted class

File: src/modules/global_css/index.js (hunk 6 above)
Comment (EhsanKia) [inline src/modules/global_css/index.js]: I do, theatrechange sends an event with state = false

**Pull request merged**